### ISSUE TITLE
The URL to the map can't be /KitchenSink/map

### ISSUE THREAD

**Issue by warpech** [Starcounter/KitchenSink#10]
I would like to change the URL to the page about `<google-map>` from:

```
http://localhost:8080/KitchenSink/Geo
```

to:

```
http://localhost:8080/KitchenSink/Map
```

but then it stops working:

<img width="1431" alt="screen shot 2015-12-09 at 10 45 56" src="https://cloud.githubusercontent.com/assets/566463/11681909/0ee3788a-9e62-11e5-86ef-9d831626533c.png">

Results in exactly the same problem as:

https://github.com/Starcounter/Starcounter/issues/3126

**Comment by aloscha** [comment]
Indeed. The problem was with naming url MAP. After changes in https://github.com/StarcounterSamples/KitchenSink/tree/Map_Link_%2310 to GEO, everything works like a charm

**Comment by warpech** [comment]
Merged, thanks!

**Comment by warpech** [comment]
But this doesn't solve the orignal problem that I want the url to be `.../map`, not some workaround word. I have pushed the code that reproduces the problem to a new branch: https://github.com/StarcounterSamples/KitchenSink/tree/Starcounter_issue_3126

Steps to reproduce:

1. Build and start app 
2. Go to `http://localhost:8080/KitchenSink` in the web browser
3. Click on the "Map" link

Expected: 

Should morph to the desired URL (`http://localhost:8080/KitchenSink/Map`)

Actual:

Throws error in the Chrome console, because wrong JSON response is returned from the server.

**Comment by PatrykSzwer** [comment]
'/kitchensink/map` is working as expected now. Closing as fixed.

**Comment by warpech** [comment]
There is no page with URI `/kitchensink/map`.

We have `/kitchensink/geo` for the reason that `/kitchensink/map` did not work.

Could you please change the URI to `/kitchensink/map` and check if it works?

**Comment by PatrykSzwer** [comment]
Yes It's working. 
I'll create PR for that change.
